packages/webapp/src/elections/api/eden-contract.ts

```diff
@@ -81,12 +82,11 @@ export const getMemberGroupFromIndex = (
 export const getMemberGroupParticipants = async (
     memberAccount?: string,
     roundIndex?: number,
-    config?: ActiveStateConfigType
+    config: ActiveStateConfigType = {
+        num_participants: 13,
+        num_groups: 1,
+    }
```

Comment:
Should we turn this default into a named constant? It might bring clarification as to why _this_ default.